client/my-sites/site-settings/wrap-settings-form.jsx

```diff
@@ -56,13 +66,11 @@ const wrapSettingsForm = getFormSettings => SettingsForm => {
 					nextProps.clearDirtyFields();
 					nextProps.markSaved();
 				} else {
-					let text;
-					switch ( nextProps.saveRequestError.error ) {
+					let text = nextProps.translate( 'There was a problem saving your changes. Please try again.' );
+					switch ( nextProps.siteSettingsSaveError ) {
 						case 'invalid_ip':
 							text = nextProps.translate( 'One of your IP Addresses was invalid. Please try again.' );
 							break;
-						default:
```

Comment:
It seems this was better when it was the default, so that we weren't calling translate only to replace it if the error is `invalid_ip` (`translate` is non-trivial).